jekyll/_cci2/github-integration.adoc

```diff
@@ -323,7 +323,7 @@ Key functionality enabled through the GitHub App integration includes the follow
 * A **more flexible trigger system**, with each pipeline having any number of VCS or non-VCS triggers. This includes:
 ** **Non-repo based triggers**: xref:webhooks#custom-webhooks[Custom webhooks] enable triggering builds from any system that can emit webhook events. Refer to our link:https://discuss.circleci.com/t/product-update-trigger-pipelines-from-anywhere-custom-webhooks/49864[community forum] for an example and known limitations.
 ** **Cross-repo triggers**: Events in one repository can trigger builds on one or many other repositories. 
-** [Coming soon] **More GitHub events as triggers**: Pipelines can be set up to run on events other than "push", with more powerful customization of trigger conditions. 
+** **More GitHub events as triggers**: Pipelines can be set up to run on events other than "push", including pull request events, with more powerful customization of trigger conditions. 
```

Comment:
Could we add a link from "more github events as triggers" to https://circleci.com/docs/github-app-trigger-options/ ?